miden/Cargo.toml

```diff
@@ -35,7 +35,6 @@ harness = false
 [[test]]
 name = "miden"
 path = "tests/integration/main.rs"
-required-features = ["internals"]
```

Comment:
optional nit: If we don't need the feature here, then it probably means we no longer need the feature exposed by the crate it self.

Reply:
good catch. I've removed the feature. See latest update.